### PR TITLE
Utilize Go-BLS in Prysm

### PR DESCRIPTION
**Pull request by nisdas**
This is a part of #1366

- [x] Removes redundant bls package.
- [x] Fixes references in `keystore`.
- [x] Fix tests

### CODE REVIEW

**Comment by terenc3t** [comment]
in **Pryzm** or just for ** keystore**? What about replacing the current stubs with go-bls? will that be different PRs?

**Comment by nisdas** [comment]
@terenc3t  This just fixes the references in Prysm to `Go-BLS` from the bls wrapper package we had before. Replacing the current stubs are out of scope for this pr.

File: shared/keystore/key.go [hunk 1]
```diff
@@ -128,24 +128,26 @@ func (k *Key) UnmarshalJSON(j []byte) (err error) {
 		return err
 	}
 
-	k.PublicKey.UnBufferPublicKey(pubkey)
-	k.SecretKey.UnBufferSecretKey(seckey)
+	if err := k.PublicKey.Deserialize(pubkey); err != nil {
+		return err
+	}
+	if err := k.SecretKey.SetLittleEndian(seckey); err != nil {
+		return err
```
Comment (rauljordan) [inline shared/keystore/key.go]: Let's log more verbose errors here using fmt

File: shared/keystore/key.go [hunk 2]
```diff
@@ -129,10 +129,10 @@ func (k *Key) UnmarshalJSON(j []byte) (err error) {
 	}
 
 	if err := k.PublicKey.Deserialize(pubkey); err != nil {
-		return err
+		return fmt.Errorf("Unable to deserialize public key: %v", err)
```
Comment (prestonvanloon) [inline shared/keystore/key.go]: Do not capitalize errors

Comment (nisdas) [reply shared/keystore/key.go]: i think that was in an old commit, its not capitalized now

**Pull request merged**